contracts/QuestFactory.sol

```diff
@@ -48,6 +48,7 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
     mapping(address => NftQuestFees) public nftQuestFeeList;
     uint16 public referralFee;
     address public sablierV2LockupLinearAddress;
+    mapping(address => address) mintFeeRecipientList;
```

Comment:
It looks like we only ever look-up the mapping for the `questCreator`, and that only gets set once. I think we're better off just setting a single recipient address on a per-quest-basis during construction, or immediately after construction.